src/main/java/org/fluentd/logger/sender/RawSocketSender.java

```diff
@@ -52,6 +52,10 @@
 
     private String name;
 
+    private String host;
```

Comment:
`server` can be removed since the instance variable in the following line can be replaced with `socket.getInetAddress()`.
```
LOG.error("Cannot send logs to " + server);
```